rocketpy/environment/environment.py

```diff
@@ -396,15 +416,57 @@ def set_date(self, date, timezone="UTC"):
 
         Parameters
         ----------
-        date : Datetime
-            Datetime object specifying launch date and time.
+        date : array, Datetime
+            Array of length 4, stating (year, month, day, hour) in the
+            time zone of the parameter ``timezone``. See Notes for more
+            information.
+            Alternatively, can be a Datetime object specifying launch
+            date and time.
         timezone : string, optional
             Name of the time zone. To see all time zones, import pytz and run
-            print(pytz.all_timezones). Default time zone is "UTC".
+            ``print(pytz.all_timezones)``. Default time zone is "UTC".
 
         Returns
         -------
         None
+
+        Notes
+        -----
+        - If the ``date`` is given as an array, it should be in the same
+          time zone as specified by the ``timezone`` parameter. This local
+          time will be available in the attribute :attr:`Environment.local_date`
+          while the UTC time will be available in the attribute
+          :attr:`Environment.datetime_date`.
+
+        - If the ``date`` is given as a ``Datetime`` object without a time zone,
+          it will be assumed to be in the same time zone as specified by the
+          ``timezone`` parameter. However, if the ``Datetime`` object has a time
+          zone specified in its ``tzinfo`` attribute, the ``timezone``
+          parameter will be ignored.
+
+        Examples
+        --------
+
+        Let's set the launch date as an array:
```

Comment:
Be careful, what you call an array is actually called "list" in python

Reply:
Good catch, changed to `list` in [9c1a461](https://github.com/RocketPy-Team/RocketPy/pull/565/commits/9c1a4619ee2beb273d81f63fa78343dfa84ca790).